src/types/User.ts

```diff
@@ -45,8 +45,12 @@ export interface User {
 }
 
 export enum DefaultUserStatus {
-  online = "Online",
-  busy = "Busy",
+  isNoob = "is a n00b",
+  isAFK = "is AFK",
+  isChatting = "is chatting",
+  isExploring = "is exploring",
+  isSpectating = "is spectating",
+  isHereForIt = "is here for it",
 }
```

Comment:
This PR is being made against `staging`, so it should be using our standard statuses here in the code, not the `env/github` environment specific overrides.

Reply:
Changes reverted in: https://github.com/sparkletown/sparkle/pull/1532/commits/d641f68f108a8c6b7533774994b704d5f93b84eb